lib/slax/poker.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Slax.Poker do
+  use Slax.Context
+  alias Slax.{Github, Poker.Round}
+
+  def start_round(channel_name, issue) do
+    repo_and_issue = Regex.replace(~r".*?(\w+)/(\w+)/issues/(\d+)$", issue["url"], "\\1/\\2/\\3")
+
+    %Round{}
+    |> Round.changeset(%{
+      channel: channel_name,
+      issue: repo_and_issue
+    })
+    |> Repo.insert()
+
+    labels = Enum.map_join(issue["labels"], ", ", & &1["name"])
+    pr = if Map.has_key?(issue, "pull_request"), do: "(PR) "
+
+    response = """
+      Planning poker for #{repo_and_issue}.
```

Comment:
but what about i18n? This should use gettext for all of our spanish speaking employees no? :trollface: